packages/app/src/cli/services/dev.ts

```diff
@@ -159,6 +160,7 @@ async function prepareForDev(commandOptions: DevOptions): Promise<DevConfig> {
     partnerUrlsUpdated,
     graphiqlPort,
     graphiqlKey,
+    consistentDev: developerPlatformClient.clientName === 'app-management',
```

Comment:
I had to do a similar check for dev stores list view, so I added `ClientName` enum in `developer-platform-client`. We can probably do something like `ClientName.AppManagement` instead of hardcoding the string here

Reply:
Makes sense! changed :)